app/src/main/java/openfoodfacts/github/scrachx/openfood/models/BottomScreenCommon.java

```diff
@@ -8,6 +8,9 @@
 import org.json.JSONObject;
 
 public class BottomScreenCommon {
```

Comment:
We could move this out of /models/

Reply:
Do you have a suggestion? This is the first time I actually had more than a quick glance at this project, so that I'm not quite sure about the app's architecture. I'd probably add this to some kind of ViewModel or some Service.

Reply:
Move it to utils package. 👍